example/src/main/java/org/apache/rocketmq/example/batch/SplitBatchProducer.java

```diff
@@ -22,34 +22,44 @@
 import java.util.List;
 import java.util.Map;
 import org.apache.rocketmq.client.producer.DefaultMQProducer;
+import org.apache.rocketmq.client.producer.SendResult;
 import org.apache.rocketmq.common.message.Message;
 
 public class SplitBatchProducer {
 
+    public static final String PRODUCER_GROUP = "BatchProducerGroupName";
+    public static final String DEFAULT_NAMESRVADDR = "127.0.0.1:9876";
+
+    public static final int MESSAGE_COUNT = 100 * 1000;
+    public static final String TOPIC = "BatchTest";
+    public static final String TAG = "Tag";
+
     public static void main(String[] args) throws Exception {
 
-        DefaultMQProducer producer = new DefaultMQProducer("BatchProducerGroupName");
+        DefaultMQProducer producer = new DefaultMQProducer(PRODUCER_GROUP);
+        //You need to set namesrvAddr to the address of the local namesrv
```

Comment:
May cause the official website quick start to set namesrv through the command line to fail.

![image](https://user-images.githubusercontent.com/14222167/169222247-c94c3a0d-2a74-4a72-9253-afdaf04e313c.png)